### ISSUE TITLE
HatchStyle.Max is incorrect

### ISSUE THREAD

**Issue by hughbe** [dotnet/corefx#24167]
In the docs for HatchStyle, `HatchStyle.Max` is documented as "Specifies hatch style SolidDiamond."
https://msdn.microsoft.com/en-us/library/system.drawing.drawing2d.hatchstyle(v=vs.110).aspx

However, in [corefx](https://github.com/dotnet/corefx/blob/c0f506dc24f9c682fd69441b4fb3259b1a5f55d3/src/System.Drawing.Common/src/System/Drawing/Drawing2D/HatchStyle.cs#L65), it is defined as `LargeGrid`

Is this a bug in corefx/.NET framework, or the documentation? I think the value of `Max` should be fixed to match the value in GDI+, and to actually be the largest value of `HatchStyle`

**Comment by mellinoe** [comment]
I don't think we can feasibly change the mapping of `HatchStyle.Max` (compat and all that), so we should just treat this as a documentation bug. The docs should probably point out that `HatchStyleMax` is defined as `HatchStyleTotal - 1` in GDI+, and our version doesn't match that.

**Comment by safern** [comment]
@karelz who takes care of the documentation bugs?

**Comment by karelz** [comment]
You can submit PR in https://github.com/dotnet/docs repo - it's fairly easy. Repo owners can help if you're in doubt.
If you have questions ahead of time, @mairaw can help as well.

**Comment by safern** [comment]
Ok, thank you very much!

**Comment by mairaw** [comment]
You can just click edit on this page on docs (https://docs.microsoft.com/en-us/dotnet/api/system.drawing.drawing2d.hatchstyle) and it will take you to the right file on GitHub. Let me know if you need any help!

**Comment by safern** [comment]
Nice! thanks @mairaw
